web-app/src/pages/Dashboard/container.js

```diff
@@ -198,6 +180,55 @@ const DashboardContainer = Wrapped =>
       });
     };
 
+    handleFormStatus(name, value, formIsValid) {
+      const formData = { ...this.state.booking.formData };
+      formData[name] = value;
+
+      if (name == 'startDate' || name == 'endDate') {
+        formData.isHalfday = false;
+        formData.isWFH = false;
+      }
+
+      if (
```

Comment:
This if block needs tidied up a little, the styling of the syntax isn't consistent.